### PR TITLE
Replace gst-plugins-base with gstreamer

### LINKED ISSUE #89: gst-plugins-base dependency blocks linux package
There is a problem pulling in the latest version due to the use of the now deprecated `gst-plugins-base-feedstock` https://github.com/conda-forge/gst-plugins-base-feedstock/issues/13. 

```
UnsatisfiableError: The following specifications were found to be in conflict:
  - cairo=1.16
  - qt==5.6.2=hf516382_1011 -> gst-plugins-base[version='>=1.12.5,<1.13.0a0'] -> gstreamer[version='>=1.12.5,<1.13.0a0'] -> glib[version='>=2.56.2,<2.57.0a0']
```
I think this is fixed by replacing `gst-plugins-base` by `gstreamer`.

### PR DESCRIPTION
**Pull request by croth1**
Trying to fix #89.

### CODE REVIEW

**Comment by ccordoba12** [comment]
I don't understand the error you posted above. Where does it appear? and why is it relevant to this PR?

**Comment by croth1** [comment]
@ccordoba12 those are the last lines in the azure-pipeline CI check. But I am not sure whether this is relevant at all.

**Comment by croth1** [comment]
@conda-forge/qt since the azure failure seems to be testing code and irrelevant here, this should be ready for review!

File: recipe/meta.yaml [hunk 1]
```diff
@@ -23,7 +23,7 @@ source:
     - 0001-shobjidl-Fix-compile-guard-around-SHARDAPPIDINFOLINK.patch  # [win]
 
 build:
-  number: 1011
+  number: 12
```
Comment (mariusvniekerk) [inline recipe/meta.yaml]: if you're not doing a new version you probably want to keep this at 1012

**Comment by mariusvniekerk** [comment]
The azure thing can probably be ignored for now, but it does indicate that we may be building qtwebkit wrong somewhere

**Comment by ccordoba12** [comment]
Ok, let's merge this.

**Pull request merged**